### DISCUSSION TITLE
Applying styles to all subdomains?

### DISCUSSION THREAD

**Discussion by ghost** [openstyles/stylus#1214] [Q&A] [answered]
I am trying to write a style that falls under all subdomains of wikipedia.org, ex: "pt.wikipedia.org", "fr.wikipedia.org", ect...

When I apply it manually to a subdomain, it works as expected, but there is of course a limit to how many i can add. Is there some sort of umbrella way to do this? I have tried "*.wikipedia.org" and the various URL applied to settings.

**Comment by tophf** [comment, marked as answer]
Use `wikipedia.org` so it will apply to all subdomains automatically.
In the CSS code it will be `domain("wikipedia.org")`.
![image](https://user-images.githubusercontent.com/1310400/111571747-55e90980-87b8-11eb-8fc5-b62f1ab35837.png)

An alternative method (not recommended) is a regexp like `^https?://[-\w]+\.wikipedia\.org/.*`
